doc/reference/aliases/install.rst

```diff
@@ -0,0 +1,6 @@
+@install
+========
+
+This alias depends on the ``*.install`` files used by the :doc:`opam
```

Comment:
While "depends" is correct, I think a more understandable high-level description would be saying that this will create the install files which in turn...